jobs/commons/src/services/read-model.service.ts

```diff
@@ -0,0 +1,39 @@
+import { Db, MongoClient, MongoClientOptions, ReadPreferenceMode } from 'mongodb'
+import { ReadModelConfig } from '../index.js'
+
+export class ReadModelClient {
+  private mongodbClient: MongoClient
+  private database: Db
+
+  private constructor(mongodbClient: MongoClient, db: Db) {
+    this.mongodbClient = mongodbClient
+    this.database = db
+  }
+
+  /**
+   * Connects to the mongodb database
+   */
+  public static async connect(config: ReadModelConfig) {
+    const connectionConfig = {
+      replicaSet: config.mongodbReplicaSet,
+      directConnection: config.mongodbDirectConnection,
+      readPreference: config.mongodbReadPreference as ReadPreferenceMode,
+      retryWrites: config.mongodbRetryWrites,
+    } satisfies MongoClientOptions
+
+    const connectionString = `mongodb://${config.readModelDbUser}:${config.readModelDbPassword}@${config.readModelDbHost}:${config.readModelDbPort}`
+    const mongodBClient = await new MongoClient(connectionString, connectionConfig).connect()
+    return new ReadModelClient(mongodBClient, mongodBClient.db(config.readModelDbName))
+  }
+
+  db() {
+    return this.database
+  }
```

Comment:
May I propose to return the db like this?

```suggestion
 get db() {
    return {
      eservices: this.database.collection<{ data: EService }>('eservices'),
      attributes: this.database.collection<{ data: Attribute }>('attributes'),
      // ...
    }
  }
```

Reply:
This would allow the `ReadModelClient` to be used like this:

```typescript
readModel.db.eservices.find({ ... })
```

Or maybe we could get rid of the `db` property and make it like this:

```typescript
readModel.eservices.find({ ... })
```


I find it to be a more precise abstraction of the read model, the only con I see it's the fact that we need to hard code the collection names (maybe in the `read-model.config` file). 
I would not put env dependencies in the commons package. Unless we don't build a proper "env injection" system

Reply:
I like the idea!
Collections names may be passed in the `config` parameter that contains all other connection settings. But that means that the deployment of every service must include all collections names, even if that service/job does not use them.
I'm seriously thinking about your proposal of hardcoding the names in the lib.

Let me sleep on it 😄

Reply:
done!